transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollDomainSocketChannel.java

```diff
@@ -37,6 +37,13 @@ public EpollDomainSocketChannel(Channel parent, FileDescriptor fd) {
         super(parent, fd.intValue());
     }
 
+    /**
+     * Creates a new {@link EpollDomainSocketChannel} from an existing {@link FileDescriptor}
+     */
+    public EpollDomainSocketChannel(FileDescriptor fd) {
```

Comment:
Yeah, then let's leave it as it is and then move `FileDescriptor` and `NativeFileDesciptor` to `netty-transport-native-common`. I'd say it's even better merging the two classes into one.